docs/preparing_datasets.md

```diff
@@ -0,0 +1,20 @@
+Preparing Datasets
+==================
+
+## Mortgage Example
+1. Setup [Apache Toree Jupyter notebook](/docs/toree.md).
+2. Download raw data from: https://rapidsai.github.io/demos/datasets/mortgage-data
+3. Run [Mortgage ETL job](https://github.com/rapidsai/spark-examples/blob/master/notebook/ETL/MortgageETL.ipynb).
+
+## Taxi Example
+1. Setup [Apache Toree Jupyter notebook](/docs/toree.md).
+2. Install `cudatoolkit` and `numba` (`conda` example provided, but you can also use `pip`):
+```bash
+conda install numba
+conda install cudatoolkit
+```
+3. Download raw data:
+```bash
+wget https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_20{09..16}-{01..12}.csv
```

Comment:
where do i put downloaded data?

Reply:
I have updated the notebook to have clearer INPUT and OUTPUT paths. Hopefully that covers it. (and the next)